pkg/volume/rbd/rbd_util.go

```diff
@@ -137,9 +138,13 @@ func (util *RBDUtil) rbdLock(b rbdMounter, lock bool) error {
 		// for defencing, get the locker name, something like "client.1234"
 		cmd, err = b.plugin.execCommand("rbd",
 			append([]string{"lock", "list", b.Image, "--pool", b.Pool, "--id", b.Id, "-m", mon}, secret_opt...))
-		output = string(cmd)
-		glog.Infof("lock list output %q", output)
+		if cmd == nil {
```

Comment:
my understanding is that cmd will never be nil even if rbd is not found, isnt that so ?